oauth2_http/javatests/com/google/auth/oauth2/ComputeEngineCredentialsTest.java

```diff
@@ -296,7 +295,27 @@ public void sign_sameAs() throws IOException {
     ComputeEngineCredentials credentials =
         ComputeEngineCredentials.newBuilder().setHttpTransportFactory(transportFactory).build();
 
-    assertArrayEquals(expectedSignature, credentials.sign(expectedSignature));
+    assertEquals(defaultAccountEmail, credentials.getAccount());
```

Comment:
I think we want to still test the signature in this test. Perhaps keep both assertions or split the `getAccount()` test into its own test.

Reply:
editing glitch. fixed.